### ISSUE TITLE
The links in readme are all broken

### ISSUE THREAD

**Issue by shatanikmahanty** [rodydavis/plugins#90]
None of the links given in readme file is opening the plugin specific page for GitHub

**Comment by rodydavis** [comment]
Yeah just saw that today, i'm going through them and updating them. Thank you!
